src/main/java/net/starschema/clouddb/jdbc/BQConnection.java

```diff
@@ -304,6 +311,19 @@ private static Integer parseIntQueryParam(String param, @Nullable String paramVa
     return val;
   }
 
+  /**
+   * Return null if {@code string} is null. Otherwise, return an array of delegates iff {@code
+   * string} can be parsed as an array when split by {@code delimiter}.
+   */
+  private static List<String> parseArrayQueryParam(@Nullable String string, Character delimiter)
+      throws BQSQLException {
+    List<String> delegates = null;
```

Comment:
While this driver code is mixed on usage, there's a best practice of never returning null lists, but instead returning empty lists.  See Effective Java 3rd edition, item 54 "Return empty collections or arrays, not nulls".